validphys2/src/validphys/compareinconsistentclosuretemplates/multiclosure_analysis.yaml

```diff
@@ -0,0 +1,242 @@
+meta:
+  title: Multiclosure report
+  author: AB
+  keywords: [inconsistencies, multiclosure]
+
+#COMPAREFITS SETTINGS
+
+positivity:
+  from_: fit
+
+description:
+  from_: fit
+
+dataset_inputs:
+  from_: fit
+  
+t0_info:
+  - use_t0: True
+    datacuts:
+      from_: fit
+    t0pdfset:
+      from_: datacuts 
+compare_settings:
+  current:
+    fit: {id: "240210_mnc_dis_ict_lam02"}
+    pdf: {id: "240210_mnc_dis_ict_lam02", label: "Current Fit"}
+    theory:
+      from_: fit
+    theoryid:
+      from_: theory
+    speclabel: "Current Fit"
+  
+  reference:
+    fit: {id: "240210_mnc_dis_ict_lam04"}
+    pdf: {id: "240210_mnc_dis_ict_lam04", label: "Reference Fit" }
+    theory:
+      from_: fit
+    theoryid:
+      from_: theory
+    speclabel: "Reference Fit"
```

Comment:
Hi @andreab1997 , a weird thing here is that we can basically compare as many fits (see `lambdavalues` below) as we want in terms of pca_template, single_point_template, etc..

However, correct me if I am wrong, the part of this script that does `vp-comparefits` only compares two.

So if the above is right, I think that it would be nicer to have a `multi vp-comparefits` (meaning that more than 2 fits can be compared in principle) feature for this script.